core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -102,3 +103,9 @@ case object KAFKA_0_10_0_IV0 extends ApiVersion {
   val messageFormatVersion: Byte = Message.MagicValue_V1
   val id: Int = 4
 }
+
+case object KAFKA_0_10_0_IV1 extends ApiVersion {
+  val version: String = "0.10.0-IV1"
+  val messageFormatVersion: Byte = Message.MagicValue_V1
+  val id: Int = 4
```

Comment:
Yeah, that's the downside of that approach. I thought that we could take advantage of the fact that we're making the change right before the release, but maybe that's confusing. I vote for keeping it simple and going with the IV approach as in the initial PR.